sqitch/Dockerfile

```diff
@@ -1,8 +1,19 @@
-FROM matteofigus/docker-sqitch
```

Comment:
FYI, this is the Dockerfile there: https://github.com/matteofigus/docker-sqitch/blob/8d6d4e72b5a677e4a93a03f44e5f0b8ad71c390e/Dockerfile